microsetta_interface/templates/admin_ffq_codes.jinja2

```diff
@@ -13,13 +13,82 @@
         <br /><br />
     {% endif %}
     <hr />
+
     <form method="post" enctype="multipart/form-data">
         {{ _('Generate Multiple Codes and download CSV File') }}<br />
         <label for="code_quantity">{{ _('Number of Codes') }}:</label>
         <input type="text" name="code_quantity" size="4">
         <input type="submit" name="generate_csv" value="{{ _('Generate') }}">
     </form>
     <hr />
-<div class="container">
-</div>
+
+    <form method="get" action="/admin/ffq_codes/search">
+        {{ _('Search FFQ Codes') }}<br />
+        <label for="email_query">{{ _('Email') }}:</label>
+        <input type="text" name="email_query">
+        <input type="submit" value="{{ _('Search') }}">
+    </form>
+
+    {% if ffq_diagnostics is not none and ffq_diagnostics|length > 0 %}
+    <div class="container">
+        <div class="list-group">
+            <div class="row">
+                <div class="col-sm">
+                    <i>{{ _('Email') }}</i>
+                </div>
+                <div class="col-sm">
+                    <i>{{ _('Transaction Created Time') }}</i>
+                </div>
+                <div class="col-sm">
+                    <i>{{ _('FFQ Registration Code') }}</i>
+                </div>
+                <div class="col-sm">
+                    <i>{{ _('Registration Code Used') }}</i>
+                </div>
+                <div class="col-sm">
+                    <i>{{ _('Registration Code Used Time') }}</i>
+                </div>
+            </div>
+            {% for code_obj in ffq_diagnostics %}
```

Comment:
On this output, would it be possible to ensure the columns  stay in alignment for all records? Some of the the test emails I use push the extremes, but it would be good to make sure the output gracefully handles as much as it possibly can.

![ffq_lookup](https://github.com/biocore/microsetta-interface/assets/83246693/43587ad0-0cd2-49be-9100-ea393cac7853)

Reply:
Pushed another commit that solves this issue. 

Note: I've also found similar overflow behavior in `admn_interested_users.jinja2` so I applied the change to that email column as well. I'm not sure if it's needed on the 2 name columns, so only 1 column was adjusted in this file.